### PR TITLE
'solution'

### PR DESCRIPTION
**Pull request by b-antonenko**

### CODE REVIEW

File: js.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+class Numberdrome {
+    constructor() {
```
Comment (gestap) [inline js.js]: use 2 whitespaces instead of 4

File: index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<script src="js.js"></script>
+</body>
+</html>
```
Comment (gestap) [inline index.html]: the document must end with one empty line

File: js.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        this.list = [];
+    }
+
+    addNumber(n) {
+        this.list.push(n);
+        return this.list;
```
Comment (gestap) [inline js.js]: we need only add the number without returning it

File: js.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    removeNumber(n) {
+        this.list.splice(this.list.indexOf(n), 1)
+        return this.list;
```
Comment (gestap) [inline js.js]: we need only remove the number without returning it

File: js.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    sum() {
+        let sumTotal = 0;
+        if (this.list.length !== 0) {
+        for (let i=0; i<= this.list.length; i++) {
```
Comment (gestap) [inline js.js]:
```suggestion
        for (let i = 0; i <= this.list.length; i++) {
```

File: js.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+            sumTotal += this.list.length[i];
+            return sumTotal;
+        }};
+        return 0;
```
Comment (gestap) [inline js.js]: return `sumTotal` it equal 0

File: js.js [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    product() {
+        let prodTotal = 0;
+        if (this.list.length !== 0) {
+            for (let i=0; i<= this.list.length; i++) {
+                prodTotal *= this.list[i];
+                return prodTotal;
+            }
+        } return 1;
```
Comment (gestap) [inline js.js]: move `return` down

https://github.com/mate-academy/style-guides/blob/master/javascript.md#whitespace--after-blocks

File: js.js [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+        } else throw new Error('Mistake');
+    }
+
+    max() {
+        let maxNum;
+        if (this.list.length !== 0) {
+            return maxNum = Math.max(...this.list);
+        } else throw new Error('Mistake');
+    }
+}
```
Comment (gestap) [inline js.js]: the document must end with one empty line

File: js.js [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+            sumTotal += this.list.length[i];
+            return sumTotal;
+        }};
+        return 0;
+    }
+
+    product() {
+        let prodTotal = 0;
+        if (this.list.length !== 0) {
+            for (let i=0; i<= this.list.length; i++) {
```
Comment (gestap) [inline js.js]:
```suggestion
            for (let i = 0; i <= this.list.length; i++) {
```

**Review by ost-ptk — CHANGES_REQUESTED** [review]

File: js.js [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+  sum() {
+      let sumTotal = 0;
+      if (this.list.length !== 0) {
+      for (let i = 0; i <= this.list.length; i++) {
+          sumTotal += this.list.length[i];
+          return sumTotal;
+      }};
+
+      return sumTotal = 0;
```
Comment (ost-ptk) [inline js.js:25]:
```suggestion
      return sumTotal;
```

File: js.js [hunk 11]
```diff
@@ -0,0 +1,52 @@
+'use strict';
+
```
Comment (ost-ptk) [inline js.js:2]: use 2 whitespaces instead of 4 in all document
https://github.com/mate-academy/style-guides/blob/master/javascript.md#whitespace--spaces

**Review by ost-ptk — CHANGES_REQUESTED** [review]

File: js.js [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+      }
+    }
+
+      return sumTotal = 0;
+  }
+
+  product() {
+    let prodTotal = 0;
+    if (this.list.length !== 0) {
+      for (let i=0; i<= this.list.length; i++) {
```
Comment (ost-ptk) [inline js.js:31]:
```suggestion
      for (let i = 0; i <= this.list.length; i++) {
```

File: js.js [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    return 1;
+  }
+
+  min() {
+    let minNum;
+    if (this.list.length !== 0) {
+      return minNum = Math.min(...this.list);
+    } else throw new Error('Mistake');
```
Comment (ost-ptk) [inline js.js:44]: no need for this else

File: js.js [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+  }
+
+  max() {
+    let maxNum;
+    if (this.list.length !== 0) {
+      return maxNum = Math.max(...this.list);
+    } else throw new Error('Mistake');
```
Comment (ost-ptk) [inline js.js:51]: no need for this else

File: js.js [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+      }
+    }
+
+    return 1;
+  }
+
+  min() {
+    let minNum;
+    if (this.list.length !== 0) {
+      return minNum = Math.min(...this.list);
```
Comment (ost-ptk) [inline js.js:43]: you can return the result without this variable.
`minNum` redundant variable

File: js.js [hunk 16]
```diff
@@ -0,0 +1,53 @@
...
+    } else throw new Error('Mistake');
+  }
+
+  max() {
+    let maxNum;
```
Comment (ost-ptk) [inline js.js:48]: you can return the result without this variable.
`maxNum` redundant variable

**Review by ost-ptk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**